apps/condo/migrations/20221208114036-0198_remove_ticket_operator_and_more.js

```diff
@@ -0,0 +1,68 @@
+// auto generated by kmigrator
+// KMIGRATOR:0198_remove_ticket_operator_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMSBvbiAyMDIyLTEyLTA4IDA2OjQwCgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucwoKCmNsYXNzIE1pZ3JhdGlvbihtaWdyYXRpb25zLk1pZ3JhdGlvbik6CgogICAgZGVwZW5kZW5jaWVzID0gWwogICAgICAgICgnX2RqYW5nb19zY2hlbWEnLCAnMDE5Nl9jb250YWN0X2NvbnRhY3RfdW5pcXVlX3Byb3BlcnR5X3VuaXRuYW1lX3VuaXR0eXBlX3Bob25lJyksCiAgICBdCgogICAgb3BlcmF0aW9ucyA9IFsKICAgICAgICBtaWdyYXRpb25zLlJlbW92ZUZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSd0aWNrZXQnLAogICAgICAgICAgICBuYW1lPSdvcGVyYXRvcicsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLlJlbW92ZUZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSd0aWNrZXRjaGFuZ2UnLAogICAgICAgICAgICBuYW1lPSdvcGVyYXRvckRpc3BsYXlOYW1lRnJvbScsCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLlJlbW92ZUZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSd0aWNrZXRjaGFuZ2UnLAogICAgICAgICAgICBuYW1lPSdvcGVyYXRvckRpc3BsYXlOYW1lVG8nLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5SZW1vdmVGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ndGlja2V0Y2hhbmdlJywKICAgICAgICAgICAgbmFtZT0nb3BlcmF0b3JJZEZyb20nLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5SZW1vdmVGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ndGlja2V0Y2hhbmdlJywKICAgICAgICAgICAgbmFtZT0nb3BlcmF0b3JJZFRvJywKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuUmVtb3ZlRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3RpY2tldGhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdvcGVyYXRvcicsCiAgICAgICAgKSwKICAgIF0K
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field operator from ticket
```

Comment:
Can this operation slowdown the database during execution? Maybe add statement timeout here?

Reply:
This is the commented line (and the entire migration).
We remove tables/columns in `apps/condo/bin/fix-database-migration-state.js`

Reply:
Yeah, missed it